docs/HowToFAQ/FAQMethodSequence.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Is there a way to write `if foo().equals(bar()) { x } else { y }` where `foo` and `bar` are methods?
```

Comment:
I think this question is too specific.  What is `.equals`?  What are `x` and `y`?

Reply:
Well, that is how the questioner put it, with some pseudo code. But to your point I took out the example and left the Q's point.